graphql-protocol/src/main/java/com/linecorp/armeria/server/graphql/protocol/AbstractGraphqlService.java

```diff
@@ -164,6 +164,19 @@ protected abstract HttpResponse executeGraphql(ServiceRequestContext ctx, Graphq
         return parseJsonString(value);
     }
 
+    @Nullable
+    private static String toStringFromJson(@Nullable Object maybeString) {
+        if (maybeString == null) {
+            return null;
+        }
+
+        if (maybeString instanceof String) {
+            return (String) maybeString;
+        } else {
+            throw new IllegalArgumentException("Unknown parameter type variables");
```

Comment:
Could you check the returned status? 400 bad request seems good for the invalid `operationName` but I guess this response status is 500.